libindi/drivers/telescope/celestrondriver.cpp

```diff
@@ -761,7 +766,7 @@ bool CelestronDriver::set_track_mode(CELESTRON_TRACK_MODE mode)
 
 bool CelestronDriver::hibernate()
 {
-    return send_command("x#", 2, response, 0, true, true);
+    return send_command("x#", 2, response, 1, true, true);
```

Comment:
Can you elaborate on this? hibernate has a response? this didn't work before? or was the response ignored?

Reply:
Yes, hibernate returns a #.  I have a log showing this somewhere.